middleware/user.go

```diff
@@ -11,21 +11,35 @@ import (
 
 func UserCatcher(next echo.HandlerFunc) echo.HandlerFunc {
 	return func(c echo.Context) error {
-		xRhIdentity, ok := c.Get(h.PARSED_IDENTITY).(*identity.XRHID)
-		if !ok {
-			return fmt.Errorf("failed to fetch the identity header")
-		}
-
 		tenantId, ok := c.Get(h.TENANTID).(int64)
 		if !ok {
 			return fmt.Errorf("failed to pull tenant from request")
 		}
 
-		userID := xRhIdentity.Identity.User.UserID
-		if userID != "" {
-			user, err := dao.GetUserDao(&tenantId).FindOrCreate(userID)
+		userIDFromContext := ""
```

Comment:
Just an optional minor comment but:

```go
userIDFromContext := ""
```

and

```go
var userIDFromContext string
```

are equivalent.

Reply:
that's right 👍 I used var variant